code-design.md

```diff
@@ -0,0 +1,26 @@
+# Flowcrypt-ios Code Design
+
+## Error Handling
+
+We have extension with errorMessage property for Error.
+
+It checks if Error has some custom message using CustomStringConvertible protocol, otherwise it uses localizedDescription property.
```

Comment:
It'll be better to highlight code properties with \`\`, so it'll look like `Error`,  `CustomStringConvertible` etc

Reply:
Sounds good 👍